core/test_harness/CMakeLists.txt

```diff
@@ -0,0 +1,21 @@
+############################################################################
+# Copyright (c) 2018-2021 by the Cabana authors                            #
+# All rights reserved.                                                     #
+#                                                                          #
+# This file is part of the Cabana library. Cabana is distributed under a   #
+# BSD 3-clause license. For the licensing terms see the LICENSE file in    #
+# the top-level directory.                                                 #
+#                                                                          #
+# SPDX-License-Identifier: BSD-3-Clause                                    #
+############################################################################
+
+set(GTEST_SOURCE_DIR ${CMAKE_SOURCE_DIR}/gtest)
```

Comment:
Until #365 is resolved the gtest source should move into `Cabana/cmake/test_harness` as well.

Reply:
Moved